app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -253,4 +251,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onAdjustNameAmount(String name, int newAmount) {
```

Comment:
We should just do this in onNameAmountAdjustmentSubmitted, since it will happen immediately after it. It's not worth adding another listener.